rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/jwt/JwtTokenUtil.java

```diff
@@ -4,16 +4,15 @@
 import io.jsonwebtoken.Claims;
 import io.jsonwebtoken.Jwts;
 import io.jsonwebtoken.SignatureAlgorithm;
+import java.util.Date;
```

Comment:
\[Checkstyle\] WARNING: 'java\.util\.Date' should be separated from previous import group by one line\.